File: PHP/NumberOfSeniorCitizens.php
<?php
/**
 * @version PHP 8.2.0
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
namespace PHP;

class NumberOfSeniorCitizens {    

    /**
     * @param String[] $details
     * @return Integer
     */
    function countSeniors($details) {
        $num = 0;
        foreach ($details as $detail) {
            $age = (int)substr($detail, 11, 2);
            if ($age > 60) {
                $num++;
            }
        }
        return $num;
    }

}

File: GO/NumberOfSeniorCitizens.go
/**
 * @version GO 1.22.3
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
package GO

import "strconv"

func countSeniors(details []string) int {
	num := 0
	for _, detail := range details {
		age, _ := strconv.Atoi(detail[11:13])
		if age > 60 {
			num++
		}
	}
	return num
}
